etc/openqa/openqa.ini

```diff
@@ -67,3 +67,16 @@
 [audit]
 # disable auditing of chatty events by default
 blacklist = job_grab job_done
+
+## Configuration for reporting of issues from the WebUI in external tools, e.g.
+# bug tracker
+[external_reporting]
+product_tracker = bugzilla
+product_url_new = https://bugzilla.opensuse.org/enter_bug.cgi
+test_tracker = redmine
+test_url_new = https://progress.opensuse.org/projects/openqatests/issues/new
+
+## Mappings for bugzilla as external reporting tool from a job group to the
+# product name in bugzilla
+[product_mapping_bugzilla]
+1 = openSUSE Tumbleweed
```

Comment:
sorry, please also ignore this part for now. coolo wanted to have the UI first so that we can incorporate #884. As he already stated he likes it I know how to continue :-)